lib/thread.c

```diff
@@ -95,7 +95,7 @@ static void cpu_record_hash_free(void *a)
 static void vty_out_cpu_thread_history(struct vty *vty,
 				       struct cpu_thread_history *a)
 {
-	vty_out(vty, "%5zu %10zu.%03lu %9zu %8zu %9zu %8lu %9lu",
+	vty_out(vty, "%5zu %10zu.%03zu %9zu %8zu %9zu %8zu %9zu",
```

Comment:
I think you actually changed the wrong item there; gcc counts arguments starting with 1 at `vty`; it's complaining about `a->total_active` not `a->cpu.total % 1000` AFAICS.

(Or it's complaining about multiple things and I looked at a different warning.)

Reply:
different build I would imagine.  but this definately cleaned it up on the rerun when I went and looked at it.

Reply:
```build	10-Jun-2019 17:56:01	lib/thread.c: In function ‘vty_out_cpu_thread_history’:
build	10-Jun-2019 17:56:01	lib/thread.c:98:15: warning: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 5 has type ‘atomic_size_t {aka unsigned int}’ [-Wformat=]
build	10-Jun-2019 17:56:01	  vty_out(vty, "%5zu %10zu.%03lu %9zu %8zu %9zu %8lu %9lu",
build	10-Jun-2019 17:56:01	               ^
build	10-Jun-2019 17:56:01	lib/thread.c:98:15: warning: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 9 has type ‘atomic_size_t {aka unsigned int}’ [-Wformat=]
build	10-Jun-2019 17:56:01	lib/thread.c:98:15: warning: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 10 has type ‘atomic_size_t {aka unsigned int}’ [-Wformat=]```